### PR TITLE
prusa-slicer: 2.4.2 -> 2.5.0

### PR DESCRIPTION
**Pull request by newAM**
###### Description of changes

Release notes: https://github.com/prusa3d/PrusaSlicer/releases
diff: https://github.com/prusa3d/PrusaSlicer/compare/version_2.4.2...version_2.5.0

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by newAM** [comment]
Result of `nixpkgs-review pr 190138` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>ifcopenshell (python310Packages.ifcopenshell)</li>
    <li>python39Packages.ifcopenshell</li>
  </ul>
</details>
<details>
  <summary>20 packages built:</summary>
  <ul>
    <li>freecad</li>
    <li>gmsh</li>
    <li>inkcut</li>
    <li>inkscape-extensions.inkcut</li>
    <li>kicad-small</li>
    <li>kicad-unstable-small</li>
    <li>octavePackages.bim</li>
    <li>octavePackages.msh</li>
    <li>opencascade-occt</li>
    <li>prusa-slicer</li>
    <li>python310Packages.enamlx</li>
    <li>python310Packages.fipy</li>
    <li>python310Packages.kicad</li>
    <li>python310Packages.pythonocc-core</li>
    <li>python39Packages.enamlx</li>
    <li>python39Packages.fipy</li>
    <li>python39Packages.kicad</li>
    <li>python39Packages.pythonocc-core</li>
    <li>super-slicer</li>
    <li>super-slicer-latest</li>
  </ul>
</details>

**Comment by Baughn** [comment]
Since I think a lot of people are waiting for this, here's a slightly hacky drop-in section for getting prusa-slicer 2.5.0 right now.

```nix
   (prusa-slicer.overrideAttrs (old: {
        src = fetchFromGitHub {
          owner = "prusa3d";
          repo = "PrusaSlicer";
          sha256 = "sha256-wLe+5TFdkgQ1mlGYgp8HBzugeONSne17dsBbwblILJ4=";
          rev = "version_2.5.0";    
        };                                         
        buildInputs = old.buildInputs ++ [(        
          pkgs.opencascade-occt.overrideAttrs (old: rec {
            version = "7.6.2";                     
            commit = "V${builtins.replaceStrings ["."] ["_"] version}";
            src = fetchurl {                       
              name = "occt-${commit}.tar.gz";      
              url = "https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=${commit};sf=tgz";
              sha256 = "sha256-n3KFrN/mN1SVXfuhEUAQ1fJzrCvhiclxfEIouyj9Z18=";
            };                                     
          })                                       
        )];
      }))
```

File: pkgs/development/python-modules/pythonocc-core/default.nix [hunk 1]
```diff
@@ -12,18 +12,18 @@
...
 , rapidjson
 , smesh
-, swig
+, swig4
 }:
 
 stdenv.mkDerivation rec {
   pname = "pythonocc-core";
-  version = "7.5.1";
+  version = "7.6.2";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pythonocc-core/default.nix:20]: That should be a separate commit

Comment (newAM) [reply pkgs/development/python-modules/pythonocc-core/default.nix:20]: Done.

**Comment by thorstenweber83** [comment]
Result of `nixpkgs-review pr 190138` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>ifcopenshell (python310Packages.ifcopenshell)</li>
    <li>python39Packages.ifcopenshell</li>
  </ul>
</details>
<details>
  <summary>20 packages built:</summary>
  <ul>
    <li>freecad</li>
    <li>gmsh</li>
    <li>inkcut</li>
    <li>inkscape-extensions.inkcut</li>
    <li>kicad-small</li>
    <li>kicad-unstable-small</li>
    <li>octavePackages.bim</li>
    <li>octavePackages.msh</li>
    <li>opencascade-occt</li>
    <li>prusa-slicer</li>
    <li>python310Packages.enamlx</li>
    <li>python310Packages.fipy</li>
    <li>python310Packages.kicad</li>
    <li>python310Packages.pythonocc-core</li>
    <li>python39Packages.enamlx</li>
    <li>python39Packages.fipy</li>
    <li>python39Packages.kicad</li>
    <li>python39Packages.pythonocc-core</li>
    <li>super-slicer</li>
    <li>super-slicer-latest</li>
  </ul>
</details>

**Review by thorstenweber83 — APPROVED** [review]

**Comment by zarthross** [comment]
> Since I think a lot of people are waiting for this, here's a slightly hacky drop-in section for getting prusa-slicer 2.5.0 right now.

Here's a less hacky,  but more flakey approach to running right now:

```
nix run github:newAM/nixpkgs/prusa-slicer-2.5.0#prusa-slicer
```

**Review by zarthross — APPROVED** [review]

**Comment by risicle** [comment]
`nixpkgs-review` happy, macos 10.15 & nixos x86_64
